README.md

```diff
@@ -515,6 +516,28 @@ Another alternative is [Chuck](https://github.com/jgilfelt/chuck) which, althoug
 
 [LeakCanary](https://github.com/square/leakcanary) is a library that makes runtime detection and identification of memory leaks a more routine part of your application development process. See the library [wiki](https://github.com/square/leakcanary/wiki) for details on configuration and usage. Just remember to configure only the "no-op" dependency in your release build!
 
+### Set up continuous integration
+
+Continuous integration systems let you automatically build and test your project every time you push in updates to version control. CI also runs static code analysis tools, generates the apk files and distributes them.
+Some popular code analysis tools are:
+
+* [Lint](https://developer.android.com/studio/write/lint.html)
+* [Checkstyle](http://checkstyle.sourceforge.net/)
+* [Danger](https://github.com/danger/danger)
+* [PMD](https://pmd.github.io/)
+* [Findbugs](http://findbugs.sourceforge.net/)
+   
+There is a big variety of CI software which provide different features, like cloud or machine based, and also pricing plans. Pricing plans might be for free if your project is open-sourced.
```

Comment:
> wide variety

Reply:
What do you mean by "machine based"?

Reply:
I think he mentioned that is stored in a local server like Jenkins.

Reply:
yes, by machine based I meant in a local machine or local server. Will rephrase